app/models/github_repo.rb

```diff
@@ -18,4 +18,9 @@ def self.gather_repos(user)
       new(raw_repo)
     end
   end
+
+  def self.get_repo(repo_name, login)
+    raw_repo = GithubService.get_repo(repo_name, login)
```

Comment:
what does login stand for? not super important, but just want to make sure we're being as explicit as possible with our naming choices 👍